lib/Components/AsyncComponent.js

```diff
@@ -0,0 +1,56 @@
+import {AbstractComponent} from './AbstractComponent';
+
+export class AsyncComponent extends AbstractComponent {
+
+    render(element) {
+        if (!this.isRendered()) {
+            if (element) {
```

Comment:
`this.element = element || this.getRenderer().createComment();`
what do you think about it?